src/share/classes/sun/management/ThreadImpl.java

```diff
@@ -220,6 +220,11 @@ private boolean verifyCurrentThreadCpuTime() {
 
     public long getCurrentThreadCpuTime() {
         if (verifyCurrentThreadCpuTime()) {
+            if (WEA != null && WEA.isAllThreadAsWisp()) {
+                long[] times = new long[1];
+                WEA.getCpuTime(new long[]{Thread.currentThread().getId()}, times);
```

Comment:
can we add a fast path for current thread

Reply:
ok